Registry/Registry.EM_COMMON

```diff
@@ -2445,6 +2471,9 @@ rconfig   logical scm_force_wind_largescale namelist,scm  1     .false.     rh
 rconfig   integer scm_force_skintemp      namelist,scm  1       0           rh   "scm_force_skin"       "SCM surface forcing by skin temperature 0=no 1=yes"      ""
 rconfig   integer scm_force_flux          namelist,scm  1       0           rh   "scm_force_flux"       "SCM surface forcing by surface fluxes 0=no 1=yes"      ""
 
+# PSU-DENG SHALLOW CU VARIABLES
+state     real  WSUBSID          k        misc          1       -           -    "WSUBSID"              "LARGE-SCALE SUBSIDENCE IN DENG SHCU SCHEME"    "m s-1"
```

Comment:
Should we place all the variables related to this scheme together and remove many "PSU-DENG ShCu"?

Reply:
done